ui/app/src/components/Dialogs/RejectDialog.tsx

```diff
@@ -0,0 +1,407 @@
+        title={
+          <FormattedMessage
+            id="workflowCancellation.title" defaultMessage="Reject"
+          />
+        }
+        subtitle={
+          <FormattedMessage
+            id="workflowCancellation.subtitle"
+            defaultMessage="The following checked item(s) will be rejected."
+          />
+        }
+        onDismiss={onDismiss}
+      />
+      <DialogBody id="confirmDialogBody">
+        <Grid container spacing={3} className={classes.contentRoot}>
+          <Grid item xs={12} sm={7} md={7} lg={7} xl={7}>
+            <SuspenseWithEmptyState
+              resource={resource}
+              withEmptyStateProps={{
+                emptyStateProps: {
+                  title: (
+                    <FormattedMessage
+                      id="publishDialog.noItemsSelected"
+                      defaultMessage="There are no affected files"
+                    />
+                  )
+                },
+                isEmpty: (value) => value.length === 0
+              }}
+            >
+              <RejectDialogContentUI
+                resource={resource}
+                checkedItems={checkedItems}
+                onUpdateChecked={onUpdateChecked}
+                classes={classes}
+              />
+
+            </SuspenseWithEmptyState>
+          </Grid>
+
+          <Grid item xs={12} sm={5} md={5} lg={5} xl={5}>
+            <form>
+              <FormControl fullWidth>
+                <InputLabel className={classes.sectionLabel}>
+                  <FormattedMessage
+                    id="rejectDialog.rejectionReason" defaultMessage="Rejection Reason"
+                  />:
+                </InputLabel>
+                <Select
+                  fullWidth
+                  input={<SelectInput />}
+                  value={rejectionReason}
+                  onChange={(e) => setRejectionReason(e.target.value as string)}
+                >
+                  <MenuItem key={'NotApproved'} value={'NotApproved'}>
+                    <FormattedMessage id="rejectDialog.notApproved" defaultMessage="Not Approved" />
+                  </MenuItem>
+                  <MenuItem key={'IB'} value={'IB'}>
+                    <FormattedMessage
+                      id="rejectDialog.incorrectBranding" defaultMessage="Incorrect Branding"
+                    />
+                  </MenuItem>
+                  <MenuItem key={'Typos'} value={'Typos'}>
+                    <FormattedMessage id="rejectDialog.typos" defaultMessage="Typos" />
+                  </MenuItem>
+                  <MenuItem key={'BrokenLinks'} value={'BrokenLinks'}>
+                    <FormattedMessage id="rejectDialog.brokenLinks" defaultMessage="Broken Links" />
+                  </MenuItem>
+                  <MenuItem key={'NSOA'} value={'NSOA'}>
+                    <FormattedMessage
+                      id="rejectDialog.nsoa" defaultMessage="Needs Section Owner's Approval"
+                    />
```

Comment:
Don't think we need the keys here. 
Also, change the syntax so plain strings are double quoted and without the curly braces